include/definitions.h

```diff
@@ -36,8 +36,11 @@
 #define OLED_HEIGHT 64
 #define I2C_ADDRESS_SSD1305 0x3C
 
+#define PCF8574_NUM_DEVICES 3
 #define PCF8574_NUM_PORTS 8
-#define I2C_ADDRESS_PCF8574_IOExt2 0x20
+#define I2C_ADDRESS_PCF8574_IOExt0 0x20 // TODO: add correct address!!!
+#define I2C_ADDRESS_PCF8574_IOExt1 0x20 // TODO: add correct address!!!
+#define I2C_ADDRESS_PCF8574_IOExt2 0x20 // TODO: add correct address!!!
```

Comment:
#define I2C_ADDRESS_PCF8574_IOExt3 0x20 // TODO: add correct address!!!

Reply:
Looking at the datasheet, the configurable address range is from `0x20` to `0x28`. Setting addresses to `0x20, 0x21, 0x22` and changing the todo to: `TODO: set jumpers accordingly` 

![grafik](https://user-images.githubusercontent.com/20418060/134984681-3473a41e-4a91-4edd-96d6-0f377159b7ba.png)